qiime2/sdk/usage.py

```diff
@@ -243,10 +580,27 @@ def init_data_collection(self, ref, collection_type, *records):
         value = self._init_data_collection_(ref, collection_type, *records)
```

Comment:
The `*` here is intentional:

```
ERROR: test_use_init_collection_data (qiime2.tests.test_artifact_api.TestArtifactAPIUsage)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/admin/Projects/q2/qiime2/qiime2/tests/test_artifact_api.py", line 247, in test_use_init_collection_data
    action.examples['variadic_input_simple'](use)
  File "/Users/admin/Projects/q2/qiime2/qiime2/core/testing/examples.py", line 177, in variadic_input_simple
    ints = use.init_data_collection('ints', list, ints_a, ints_b)
  File "/Users/admin/Projects/q2/qiime2/qiime2/sdk/usage.py", line 585, in init_data_collection
    value = self._init_data_collection_(ref, collection_type, records)
  File "/Users/admin/Projects/q2/qiime2/qiime2/plugins.py", line 41, in _init_data_collection_
    t = ', '.join(sorted([i.ref for i in records]))
  File "/Users/admin/Projects/q2/qiime2/qiime2/plugins.py", line 41, in <listcomp>
    t = ', '.join(sorted([i.ref for i in records]))
AttributeError: 'tuple' object has no attribute 'ref'
```

 IIRC, you didn't want the `*` in the abstract method signature, and I removed it in 17cca13.  I can't remember why...  🤔

Reply:
I'll leave the rest of the exercise to you, but ping me if you need help. This * should be removed, because it was an oversight on the initial draft of the api.

Reply:
I think I squarshed it 🐛 .